lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -132,6 +131,9 @@ public JourneyResponse handleRequest(JourneyRequest event, Context context) {
             AuditEventUser auditEventUser =
                     new AuditEventUser(userId, ipvSessionId, govukSigninJourneyId, ipAddress);
 
+            List<String> userCriReturnCredentials =
+                    criResponseService.getUserIssuedCriResponse(userId);
```

Comment:
We might want to filter this on credentialIssuer=f2f in case we ending up adding records for other CRIs to this table. Also to avoid getting this confused with`credentials` maybe this could be made a boolean something like `userHasF2FRequest`?